### DISCUSSION TITLE
When links clicked with confirm dialogue, add copy option

### DISCUSSION THREAD

**Discussion by brendanmeyer** [zoidsh/meru#359] [Feature Requests]
Hey

I use the confirm dialogue for links when I click them.

I know there is a copy link in the right-click menu, but sometimes this copies the wrong link, or it does not detect the link for the right-click.

I would like to have the option to copy the link from the dialogue and the open link option. (note: it should still close the dialogue if "copy link" was clicked)

Thank you
Brendan

**Comment by cipriancraciun** [comment]
At least on Linux you can write you own `xdg-open` script that is on path (maybe a special `$PATH` only for GMail desktop), and there you could use something like `dmenu` or `rofi` to ask if the link should be opened or copied.  (In fact I use this script to switch between browsers, browser profiles, other applications, etc.)
